base/logger_file.go

```diff
@@ -384,3 +396,27 @@ func runLogDeletion(ctx context.Context, logDirectory string, logLevel string, s
 
 	return nil
 }
+
+// getDeletionDirAndRegexp will return the directory and a regexp matching log file and rotated patterns.
+func getDeletionDirAndRegexp(filename string) (string, *regexp.Regexp) {
+	dir, path := filepath.Split(filename)
+	lastDot := strings.LastIndex(path, ".")
```

Comment:
`path` and `filename` are the wrong way around from what I'd expect

```suggestion
func getDeletionDirAndRegexp(path string) (string, *regexp.Regexp) {
	dir, name := filepath.Split(path)
	lastDot := strings.LastIndex(name, ".")
```